cmd/config.go

```diff
@@ -130,41 +114,40 @@ func getConfig(flags *pflag.FlagSet) (Config, error) {
 	}, nil
 }
 
-// Reads a configuration file from disk.
-func readDiskConfig(fs afero.Fs) (Config, *configdir.Config, error) {
-	if configFile != "" {
-		data, err := ioutil.ReadFile(configFile)
-		if err != nil {
-			return Config{}, nil, err
-		}
-		var conf Config
-		err = json.Unmarshal(data, &conf)
-		return conf, nil, err
+// Reads a configuration file from disk and returns it and its path.
```

Comment:
well now it reads it from the provided afero.Fs ;)

Reply:
done